.github/workflows/release-v2.yaml

```
@@ -0,0 +1,145 @@
+
+    - name: Set up Go
+      uses: actions/setup-go@v5
+      with:
+        go-version: '1.21' # Adjust to your Go version
+
+    - name: Install dependencies
+      run: go mod download
+
+    - name: Install CGO dependencies for Linux
+      if: matrix.goos == 'linux'
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:11:8: Quote this to prevent word splitting

(shellcheck)